### PR TITLE
Add a fuzzer

### PR DESCRIPTION
**Pull request by ekse**

### CODE REVIEW

**Review by awakecoding — APPROVED** [review]
LGTM

File: CMakeLists.txt [hunk 1]
```diff
@@ -39,13 +39,36 @@ if(WIN32)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${C_FLAGS}")
 endif()
 
-set(CMAKE_EXPORT_DIR "lib/cmake/lizard")
```
Comment (awakecoding) [inline CMakeLists.txt]: why did you remove CMAKE_EXPORT_DIR?

Comment (ekse) [reply CMakeLists.txt]: Oops that was not intended, I put it back.
